packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -20,11 +20,11 @@ To better understand SCIM and the specific implementation of SCIM using Okta, se
 
 ### Creating users
```

Comment:
General comment on the `<ApiOperation>` element listed for all the operations: The URL given might need to start from ` /api/v1/ ` to match our other pages. Also `Users` in the URL should be all lowercase to match our other pages.